aptos-move/aptos-vm/src/transaction_validation.rs

```diff
@@ -57,11 +57,17 @@ pub static APTOS_TRANSACTION_VALIDATION: Lazy<TransactionValidation> =
         unified_prologue_name: Identifier::new("unified_prologue").unwrap(),
         unified_prologue_fee_payer_name: Identifier::new("unified_prologue_fee_payer").unwrap(),
         unified_epilogue_name: Identifier::new("unified_epilogue").unwrap(),
+
+        unified_prologue_v2_name: Identifier::new("unified_prologue_v2").unwrap(),
+        unified_prologue_fee_payer_v2_name: Identifier::new("unified_prologue_fee_payer_v2")
+            .unwrap(),
+        unified_epilogue_v2_name: Identifier::new("unified_epilogue_v2").unwrap(),
```

Comment:
if you could land the PR before enabling AA. I don't think we need v2, you can directly modify unified_xxx methods since they are never called.

Reply:
When will we enable AA?

Reply:
If I can get the reviews on time and if there are no major objections, I could land this code in v1.30 (2 weeks from now).

Reply:
I don't even know haha...will follow up later.

Reply:
Igor mentioned that dAA will be enabled in v1.29. dAA also uses this unified_prologue path. As this code will land only in v1.30, I guess we need `_v2` versions.